views/templates.pug

```diff
@@ -9,8 +9,9 @@
                 button.btn.btn-outline-secondary.btn-sm.dropdown-toggle(data-toggle="dropdown" type="button" aria-haspopup="true" aria-expanded="false")
                   span.current Short
                 ul.dropdown-menu.sources
-                  button.dropdown-item.btn.btn-light.btn-sm(data-bind="Short")
-                    | Short
+                  if storageSolution != "null"
```

Comment:
Can you change the eq operator to `!==` to make it explicit that we are checking for the string "null"? It might catch someone off-guard in the future

Reply:
Good point!